src/logic/safe/transactions/gas.ts

```diff
@@ -147,6 +147,11 @@ const getGasEstimationTxResponse = async (txConfig: {
 
     if (!sameString(result, EMPTY_DATA)) {
       return new BigNumber(result.substring(138), 16).toNumber()
+    } else {
+      // This is a transfer of native coin (eth, xDai, etc), in that case the empty result is valid
```

Comment:
As far as I investigated, the transfer of native network coin is represented as a transaction with `0x` data and `0x` response, so I added this logic to avoid throwing an error for this valid case. Could you confirm if this logic (data as `0x` for native transfers) it's ok @rmeissner ? thanks!

cc: @fernandomg

Reply:
So as this is used in combination with calling `requiredTxGas` on a safe contract `txConfig.data` should never be empty. Therefore I would say that something is not yet completely correct in this flow.